charts/app-config/values-integration.yaml

```diff
@@ -2282,10 +2277,18 @@ govukApplications:
             secretKeyRef:
               name: search-api-v2-rabbitmq
               key: RABBITMQ_URL
-        - name: GOOGLE_KEYFILE_READ
-          value: /etc/keys/google-key-read.json
-        - name: GOOGLE_KEYFILE_WRITE
-          value: /etc/keys/google-key-write.json
+        - name: GOOGLE_CLOUD_CREDENTIALS
+          value: /etc/credentials/google-cloud/credentials.json
+        - name: DISCOVERY_ENGINE_DATASTORE
+          valueFrom:
+            secretKeyRef:
+              name: search-api-v2-google-cloud-discovery-engine-configuration
```

Comment:
info, no action needed: this name is close to the max length (63) but that's ok for now cos we're not prefixing it with the chart name etc.

Reply:
It _is_ a bit of a mouthful 😬